install_config/redeploying_certificates.adoc

```diff
@@ -301,33 +317,111 @@ openshift_master_ca_certificate={'certfile': '</path/to/ca.crt>', 'keyfile': '</
 If you do not set the above, then the current CA will be regenerated in the next
 step.
 
-. Run the *_redeploy-certificates.yml_* playbook with the
-`openshift_certificates_redeploy_ca=true` variable, specifying your inventory
-file:
+. Run the *_redeploy-openshift-ca.yml_* playbook, specifying your inventory file:
 +
 ----
 $ ansible-playbook -i <inventory_file> \
-    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/redeploy-certificates.yml \
-    --extra-vars "openshift_certificates_redeploy_ca=true"
+    /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/redeploy-openshift-ca.yml
 ----
 
 . As discussed previously, the playbook now serially evacuates all nodes in the
-cluster, requiring manual redeployment of registry and router certificates. See
-xref:redeploying-registry-and-router-certificates[Redeploying Registry and
-Router Certificates] to complete this procedure.
+cluster. If you are using custom registry or router certificates, these require
+manual redeployment. See
+xref:redeploying-custom-registry-or-router-certificates[Redeploying Custom Registry or Router Certificates] to complete this procedure.
+
```

Comment:
Hmm. It also has a few nuanced differences / notes, like the different services that are restarted. Will think on it.